tools/scripts/unit_tests.pl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env perl
```

Comment:
This should be in `DDS_ROOT/bin` or the other test related files should be moved to `DDS_ROOT/tools/scripts/unit_tests.pl`.

Reply:
I agree with the latter.  All this random stuff doesn't need to be added to the user's PATH.